File: Python/practice2.py
import pymysql

config={
    'user':'sa',
    'password':'!password123',
    'host':'127.0.0.1',
    'database':'test',
}

cnx = pymysql.connect(**config)
cursor = cnx.cursor()
query=("SELECT * FROM test1 WHERE price > 600 LIMIT 10")
cursor.execute(query)

for (id,amount,price) in cursor :
	print("{},{},{}".format(id,amount,price))
	

cursor.close()
cnx.close()

File: nodejs/practice2.js
var mysql = require('mysql');

var conn = mysql.createConnection({
	host:'localhost',
	database:'test',
	user:'sa',
	password:'!password123'
});

conn.connect(
	function(err){
		if(err){
			console.log("connection failed");
		}
		else{
			console.log("connection success");
		}
	}
);

conn.query('SELECT * FROM test1 LIMIT 100',
	function(err,rows){
		if(err){
			console.log('query failed');
		}
		else{
			console.log(rows.length);
			for(i=0;i<rows.length;i++){
				console.log(rows[i].id);
			}
		}
	}
);

conn.end(
	function(err){
		if(err){
			console.log('connection end error');
		}
		else{
			console.log('connection end success');
		}
	}
)
